### ISSUE TITLE
QuestDB 6.4.2 does not load on Windows

### ISSUE THREAD

**Issue by pjsgsy** [questdb/questdb#2310]
### Describe the bug

Starts and stops immediately if installed as a service, so ran interactively to try and see what is going on. 

This is all that happens. Process quits and does not run until stopped

"D:\Data\Downloads and Apps\questdb-6.4.2-rt-windows-amd64\bin>questdb

  ___                  _   ____  ____
 / _ \ _   _  ___  ___| |_|  _ \| __ )
| | | | | | |/ _ \/ __| __| | | |  _ \
| |_| | |_| |  __/\__ \ |_| |_| | |_) |
 \__\_\\__,_|\___||___/\__|____/|____/
                        www.questdb.io

JAVA_EXE: D:\Data\Downloads and Apps\questdb-6.4.2-rt-windows-amd64\bin\java.exe

QuestDB server 6.4.2
Copyright (C) 2014-2022, all rights reserved.

Reading log configuration from qdbroot\conf\log.conf
Log configuration loaded from: qdbroot\conf\log.conf
Log configuration loaded from default internal file.

D:\Data\Downloads and Apps\questdb-6.4.2-rt-windows-amd64\bin>
"

The following is logged in the windows event log

"The description for Event ID 0 from source QuestDB cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

JAVA_HOME D:\Data\Downloads and Apps\questdb-6.4.2-rt-windows-amd64\bin\java.exe "

That's unfortunately as far as I can get!

### To reproduce

Install. Run Influxdb from command prompt

### Expected Behavior

Process should stay running and accept queries, etc.

### Environment

```markdown
Windows server 2019
```

**Comment by bziobrowski** [comment]
Hello @pjsgsy . 
Is that the only qdb event visible in Event Viewer ? 
Could you also please check logs in C:\Windows\System32\qdbroot\conf .

**Comment by pjsgsy** [comment]
Yes, the only event that is logged

I was running interactively via the command line, so not sure the log will be in system32?  But, for when I ran it as a service, yes, they are, and that log contains I think the error that it could not bind to port 9000, and as it happens, I know I have something else on that port. 

For info, I re-installed and am using the '-d d:\QuestDB' as a working folder. I just tried again.  The same error does not appear to be logged to d:\QuestDB\log files, but, if I stop the other process running on that port, it does in fact load.

So, the second challenge -   In the d:\QuestDBzconf folder, I have server.conf. The line, commented out

# IP address and port of HTTP server
#http.net.bind.to=0.0.0.0:9000

So, I change this to 

# IP address and port of HTTP server
http.net.bind.to=0.0.0.0:9001

But, on reload, I get this

Reading log configuration from d:\questdb\conf\log.conf
Log configuration loaded from: d:\questdb\conf\log.conf
Configuration issues:
    Invalid settings (not recognized, probable typos):
        * ï»¿#

I guess there are some odd characters in the file by default because I copy the whole file to the clipboard, ctrl-shift-v to paste plain text only, resave the file, and it loads!

So, progress. Thank you. I wonder why the error is only reported to the log when you run as a service, and not interactively!?

**Comment by pjsgsy** [comment]
And, if you do not mind whilst here, if I install as a service, which is what I would ultimately do, there appears to be no way to specify the -d parameter?  Thanks again.

**Comment by bziobrowski** [comment]
If you run questdb.exe without any parameters then it starts not as a service but as a standalone instance in current directory (so logs go to ~/log/ ) . 
If you want to run qdb as a service in custom directory then you need to specify -d with 'questdb.exe install' only (it's stored in windows registry afterwards).

> I guess there are some odd characters in the file

That's unlikely . I've done the same with notepad and everything worked fine . I also confirmed that file's encoding is valid UTF-8 .

**Comment by pjsgsy** [comment]
OK - I tried copying the original config file back to my install folder and replicated the process and error. All I did was remove the # and change port number to 9001. I tried and got the error first with notepad, then I tried Wordpad, in case of character set issues. Same. It was only when I tried copy all to clipboard and paste as plain test+ resave then the issue resolved.

In any case, I think you solve all my issues, so, thank you.
